modules/system/traits/AssetMaker.php

```diff
@@ -127,6 +127,21 @@ public function addJs($name, $attributes = [])
         }
 
         $jsPath = $this->getAssetScheme($jsPath);
+        // Merge with 'default' attribute(s)
+        $attributes += [
+            'cache' => true
+        ];
+
+        // If needed no cache, prevent Cloudflare Rocker Loader to add additional attributes.
+        // @See https://github.com/octobercms/october/pull/4092
+        if(!$attributes['cache']){
+            $attributes['data-cfasync'] = 'false';
+        }
+
+        // Unset unnecessery attribute(s) to be printed.
+        $attributes = array_except($attributes, [
+            'cache'
+        ]);
```

Comment:
This can be simplified to the following:

```php
// Prevent CloudFlare's Rocket Loader from breaking stuff
// @see octobercms/october#4092, octobercms/october#3841, octobercms/october#3839
if (isset($attributes['cache']) && $attributes['cache'] == false) {
    $attributes['data-cfasync'] = 'false';
    unset($attributes['cache']);
}

Reply:
Okay. Changed.
At first, I'd like to make it easy to implement for issues in the future.  But that's fine.